testutil/ciao-down/prepare.go

```diff
@@ -116,7 +119,30 @@ func prepareSSHKeys(ctx context.Context, ws *workspace) error {
 	return nil
 }
 
+func getProxy(upper, lower string) (string, error) {
```

Comment:
If we keep osprepare, it probably should leverage this as a higher level library utility function.